space-cli/cmd/modules/accounts/view.go

```diff
@@ -0,0 +1,49 @@
+package accounts
+
+import (
+	"os"
+	"strings"
+	"unicode/utf8"
+
+	"github.com/olekukonko/tablewriter"
+
+	"github.com/spaceuptech/space-cli/cmd/model"
+	"github.com/spaceuptech/space-cli/cmd/utils"
+)
+
+func listAccounts(prefix string, showKeys bool) error {
+	credential, err := utils.GetCredentials()
+	if err != nil {
+		return err
+	}
+	if len(credential.Accounts) == 0 {
+		utils.LogInfo("No account found for prefix provided")
```

Comment:
Log message is incorrect

Reply:
Should I change it to "Accounts.yaml file contains no accounts. Please make an account first!" or something else...?

Reply:
`No accounts found`.